diff --git a/node_modules/riichi/index.js b/node_modules/riichi/index.js
index b22cb02..651e749 100644
--- a/node_modules/riichi/index.js
+++ b/node_modules/riichi/index.js
@@ -315,7 +315,7 @@ class Riichi {
             this.tmpResult.oya = [ceil100(base*6)]
             this.tmpResult.ko = [ceil100(base*4)]
         }
-        this.tmpResult.ten = this.isOya ? eval(this.tmpResult.oya.join('+')) : eval(this.tmpResult.ko.join('+'))
+        this.tmpResult.ten = this.isOya ? this.tmpResult.oya.reduce((a, b) => a + b, 0) : this.tmpResult.ko.reduce((a, b) => a + b, 0)
         this.tmpResult.text += ' ' + this.tmpResult.ten + '点'
         if (this.isTsumo) {
             this.tmpResult.text += '('
